### DISCUSSION TITLE
is xeol still maintained?

### DISCUSSION THREAD

**Discussion by tomersein** [xeol-io/xeol#370] [Q&A]
Hello!
the last time a pr was merged was 3 month ago, is this open source still maintained?

**Comment by somelinuxguy** [comment]
I came here wondering the same thing.

My thoughts:

Nobody has replied to this discussion, and checking PRs you can see it's all dependabot for a year, issues are open and crusty including bugs.   So if I was a betting man: I would say no this is not maintained actively.

**Comment by tomersein** [comment]
yeap, it looks indeed EOL
https://github.com/anchore/grype/issues/2083 - working on implementing this EOL via the original tool of grype
